Index: transmission-1.93/configure
===================================================================
--- transmission-1.93.orig/configure	2010-05-01 23:38:41.000000000 +0200
+++ transmission-1.93/configure	2010-05-02 04:46:54.000000000 +0200
@@ -1064,7 +1064,7 @@
 GCONF2_LIBS
 DBUS_GLIB_CFLAGS
 DBUS_GLIB_LIBS'
-ac_subdirs_all='third-party/libevent'
+#ac_subdirs_all='third-party/libevent'
 
 # Initialize some variables set by options.
 ac_init_help=
