--- a/libtransmission/web.c	2009-05-05 02:40:08.000000000 +0200
+++ b/libtransmission/web.c	2009-05-10 15:56:18.000000000 +0200
@@ -194,7 +194,7 @@
             tr_free( str );
         }
 
-        curl_easy_setopt( easy, CURLOPT_IPRESOLVE, CURL_IPRESOLVE_V4 );
+        curl_easy_setopt( easy, CURLOPT_IPRESOLVE, CURL_IPRESOLVE_WHATEVER );
         curl_easy_setopt( easy, CURLOPT_DNS_CACHE_TIMEOUT, 360L );
         curl_easy_setopt( easy, CURLOPT_CONNECTTIMEOUT, 60L );
         curl_easy_setopt( easy, CURLOPT_FOLLOWLOCATION, 1L );
